.gitignore

```diff
@@ -37,13 +68,9 @@ coverage.xml
 node_modules/
 bower_components/
 .grunt/
+vendor/
+static/vendor/
 src/vendor/
-src/static/fonts/
```

Comment:
You were incorrect about that. They start in `src/static/fonts` and get copied to `static/fonts`. That said, they should _really_ be originating from `node_modules/cf-icons/src/fonts`.